account-test/specs/step-definitions/signin.steps.js

```diff
@@ -0,0 +1,79 @@
+//const {  defineFeature, loadFeature} = require('jest-cucumber')
+import {  defineFeature, loadFeature} from 'jest-cucumber'
+import SignInPage from '../../src/pages/signIn.page.js'
+const assert = require('assert');
+const feature = loadFeature('./specs/features/login.feature');
+const { Builder, By, Key, until } = require('selenium-webdriver')
+const { querySelector } = require('../../src/util/helpers.js')
+
+defineFeature(feature, test => {
+    let driver
+    let signInPage
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000 * 60 * 5
+    
+    beforeEach(async () => {
+      driver = await new Builder().forBrowser('firefox').build()
+        signInPage = new SignInPage(driver)     
+    })
+    
+    afterEach(async()=>{
+        driver.close();
+    })
+ 
+    test('Happy path : Successful sign in journey', ({ given, when, then }) => {
+
+        given('I am on sign in page', async() => {
+           await signInPage.open()
+        });
+
+        when(/^I provide my username as (.*)$/, async(email) => {
+            await driver.findElement(By.id(signInPage.username)).sendKeys(email);
```

Comment:
Don't think we need async await here when filling in the form only on form submission below.